$OpenBSD: patch-gnome-music_in,v 1.1.1.1 2013/09/27 10:39:48 ajacoutot Exp $

XXX push upstream
AttributeError: 'module' object has no attribute 'textdomain'

--- gnome-music.in.orig	Thu Sep 26 19:53:11 2013
+++ gnome-music.in	Thu Sep 26 19:55:16 2013
@@ -45,8 +45,12 @@ def install_excepthook():
 if __name__ == "__main__":
     install_excepthook()
 
-    locale.bindtextdomain('gnome-music', localedir)
-    locale.textdomain('gnome-music')
+
+    try:
+      locale.bindtextdomain('gnome-music', localedir)
+      locale.textdomain('gnome-music')
+    except AttributeError:
+      pass
     gettext.bindtextdomain('gnome-music', localedir)
     gettext.textdomain('gnome-music')
 
